src/pymovements/datasets/raccoons.py

```diff
@@ -0,0 +1,198 @@
+from typing import Any
+
+from pymovements.dataset.dataset_definition import DatasetDefinition
+from pymovements.dataset.dataset_library import register_dataset
+
+
+@dataclass
+@register_dataset
+class RaCCooNS(DatasetDefinition):
+    """RaCCooNS dataset :cite:p:`RaCCooNS`.
+
+    The Radboud Coregistration Corpus of Narrative Sentences (RaCCooNS) dataset consists
+    simultaneously recorded eye-tracking and EEG data from Dutch sentence reading,
+    aimed at studying human sentence comprehension and evaluating computational
+    language models. The dataset includes 37 participants reading 200 narrative sentences,
+    with eye movements and brain activity recorded to analyze reading behavior
+    and neural responses. The dataset provides both raw and preprocessed data,
+    including fixation-related potentials, enabling comparisons between cognitive
+    and neural processes.
+
+    Check the respective paper :cite:p:`RaCCooNS` for details.
+
+    Attributes
+    ----------
+    name: str
+        The name of the dataset.
+
+    has_files: dict[str, bool]
+        Indicate whether the dataset contains 'gaze', 'precomputed_events', and
+        'precomputed_reading_measures'.
+
+    mirrors: dict[str, tuple[str, ...]]
+        A tuple of mirrors of the dataset. Each entry must be of type `str` and end with a '/'.
+
+    resources: dict[str, tuple[dict[str, str | None], ...]]
+        A tuple of dataset gaze_resources. Each list entry must be a dictionary with the following
+        keys:
+        - `resource`: The url suffix of the resource. This will be concatenated with the mirror.
+        - `filename`: The filename under which the file is saved as.
+        - `md5`: The MD5 checksum of the respective file.
+
+    extract: dict[str, bool]
+        Decide whether to extract the data.
+
+    filename_format: dict[str, str]
+        Regular expression which will be matched before trying to load the file. Namedgroups will
+        appear in the `fileinfo` dataframe.
+
+    filename_format_schema_overrides: dict[str, dict[str, type]]
+        If named groups are present in the `filename_format`, this makes it possible to cast
+        specific named groups to a particular datatype.
+
+    trial_columns: list[str]
+            The name of the trial columns in the input data frame. If the list is empty or None,
+            the input data frame is assumed to contain only one trial. If the list is not empty,
+            the input data frame is assumed to contain multiple trials and the transformation
+            methods will be applied to each trial separately.
+
+    column_map: dict[str, str]
+        The keys are the columns to read, the values are the names to which they should be renamed.
+
+    custom_read_kwargs: dict[str, Any]
+        If specified, these keyword arguments will be passed to the file reading function.
+
+    Examples
+    --------
+    Initialize your :py:class:`~pymovements.PublicDataset` object with the
```

Comment:
`PublicDataset` does not exist anymore. Use 

```
:py:class:`~pymovements.dataset.Dataset`
```

Reply:
done